src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +75,18 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should render the new source to equal MapType', () => {
+    const newProps = {
+      ...props,
+      isNew: true,
+      source: 'CIEL',
```

Comment:
Don't we already have a constant for this?

Reply:
yes, we do... I'll adjust accordingly

Reply:
Done